frontend/src/components/SignIn/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { IconX } from "@tabler/icons-react";
+import clsx from "clsx";
+import { useState } from "react";
+import { useErrorBoundary } from "react-error-boundary";
+import { useForm } from "react-hook-form";
+import { Link } from "wouter";
+import ActivityOverlay from "~/components/ActivityOverlay";
+import { base64Encode } from "~/helpers/base64";
+import FormTitle from "./FormTitle";
+import Layout from "~/components/Layout";
+import styles from "./SignIn.module.css";
+import Title from "~/components/Title";
+import { api } from "~/lib/api";
+import { setErrors } from "~/lib/form";
+
+type Form = {
+  userEmail: string;
+  userPassword: string;
+};
+
+type State = {
+  /* TODO: Implement this logic after auth handler is ready */
+  isAlreadySignedIn: boolean;
+  isSignInComplete: boolean;
+};
+
+export default function SignIn() {
+  const { showBoundary } = useErrorBoundary();
+  const [state, setState] = useState<State>({ isAlreadySignedIn: false, isSignInComplete: false });
+  const { clearErrors, formState: { errors, isSubmitting }, handleSubmit, register, setError } = useForm<Form>({});
```

Comment:
Not sure if this empty object is necessary.
```suggestion
  const { clearErrors, formState: { errors, isSubmitting }, handleSubmit, register, setError } = useForm<Form>();
```